guides/release/testing/index.md

```diff
@@ -18,32 +18,45 @@ Writing tests is also a fun activity, a nice change of pace from delivering feat
 
 Every Ember app comes with [QUnit](http://qunitjs.com/) and [QUnit DOM](https://github.com/simplabs/qunit-dom). QUnit is a testing framework, and QUnit DOM is a library that helps you **write tests that are concise and readable**.
 
-To see the power of QUnit DOM, consider this code snippet. It checks whether our button component shows the right text and the right CSS attribute.
+To see the power of QUnit DOM, consider this code snippet. It checks whether our button component shows the right label and the right attributes.
 
-```javascript {data-filename=tests/integration/components/simple-button/component-test.js}
+```javascript {data-filename=tests/integration/components/simple-button-test.js}
 /*
   For simplicity, the import, module, and setup statements
   are omitted here. Our component accepts two attributes,
-  text (string) and isDisabled (boolean).
+  label (string) and isDisabled (boolean).
 */
-test('should show text', async function(assert) {
-  await render(hbs`<SimpleButton @text="Hello world!" />`);
+test('should show label', async function(assert) {
+  await render(hbs`
+    <SimpleButton
+      @label="Hello world!"
+    />
+  `);
+  let button = this.element.querySelector('button');
 
   // QUnit
-  assert.strictEqual(this.element.textContent.trim(), 'Hello world!');
+  assert.strictEqual(button.textContent.trim(), 'Hello world!');
```

Comment:
Note, `this.element` refers to the `div` container that testing creates. As a result, the assertions for "should allow disabling the button" weren't correct.